projects.yml

```diff
@@ -1,30 +1,67 @@
 projects:
+  # Core software that serves www.osm.org
+  - name: OpenStreetMap Website
+    repo: https://github.com/openstreetmap/openstreetmap-website
   - name: cgimap
     repo: https://github.com/zerebubuth/openstreetmap-cgimap
-  - name: iD
-    repo: https://github.com/openstreetmap/iD
+  # Editors
   - name: JOSM
     repo: https://github.com/openstreetmap/josm
-  - name: JXAPI
-    repo: https://github.com/iandees/xapi-servlet
+  - name: JOSM Plugins
+    repo: https://github.com/openstreetmap/josm-plugins
```

Comment:
This might be a major error source, it's a svn-to-osm replication, so all the author data is fictional